cs/ccxt/base/Exchange.cs

```diff
@@ -511,9 +511,9 @@ public string totp(object a)
         return "";
     }
 
-    public bool checkAddress(object address)
+    public List<string> unique(object obj)
```

Comment:
this method was missing from `c#` base. though, it's only used in `checkAddress` method